src/react-native/13-making-http-requests/making-http-requests.md

```diff
@@ -87,11 +91,15 @@ It will be helpful to have a third terminal tab for the `npm run api` command.
 
 @sourceref ../../../exercises/react-native/13-http-requests/01-problem/src/components/Loading/Loading.test.tsx
 
-### Exercise
+✏️ Update **src/screens/StateList/StateList.test.tsx** to be:
+
+@diff ../../../exercises/react-native/12-navigation-params/01-solution/src/screens/StateList/StateList.test.tsx ../../../exercises/react-native/13-http-requests/01-problem/src/screens/StateList/StateList.test.tsx only
+
+### Exercise 1
 
 TODO
 
-### Solution
+### Solution 1
```

Comment:
part of the solution for `src/services/pmo/restaurant/hooks.ts` requires adding `const baseUrl = process.env.PMO_API`

should this be part of the initial code or are they expected to know how to pull from env variables?

![Screenshot 2024-05-10 at 11 38 09 AM](https://github.com/bitovi/academy/assets/9858612/d6fcc4e9-d76f-4514-8c7b-80dfe197cb69)

Reply:
In the lesson we will teach them how to pull from env variables, but I don't mind putting it in the initial code. LMK